Curs12/C/03-params_sum.c

```diff
@@ -0,0 +1,46 @@
+/* Programarea Calculatoarelor CA, 2022-2023
+ *
+ *	Suma elementelor parametrilor liniei de comanda
+ *
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+#include <ctype.h>
+
+bool isnumber(const char * str)
+{
+	for (int i = 0; str[i] != '\0'; i++)
+		if (!isdigit(str[i]))
+			return false;
+
+	return true;
+}
+
+int main(const int argc, const char **argv)
```

Comment:
Sorry, I changed my mind. Could you please have 

```suggestion
int main(int argc, char *argv[])
```

Reply:
why this change?

Reply:
The [C standard](https://www.open-std.org/jtc1/sc22/wg14/www/docs/n2310.pdf), per chapter 5.2.2.1, allows any equivalent of `int main(int argc, char *argv[])`, including `int main(int argc, char **argv)`, but with no reference to the use of `const` or other modifiers. 

Thus, I would recomand either `int argc, char *argv[]` or `int argc, char **argv[]`.

Reply:
The [C standard](https://www.open-std.org/jtc1/sc22/wg14/www/docs/n2310.pdf), per chapter 5.1.2.2.1 (page 11), allows any equivalent of `int main(int argc, char *argv[])`, including `int main(int argc, char **argv)`, but with no reference to the use of `const` or other modifiers. 

Thus, I would recomand either `int argc, char *argv[]` or `int argc, char **argv[]`.